app/views/devise/sessions/new.html.erb

```diff
@@ -1,26 +1,35 @@
-<h2>Log in</h2>
+<div class="row justify-content-center mt-5">
+	<div class="col-4">   
+		<div class="card mb-3 shadow " >
```

Comment:
Careful with the indentation here.

Reply:
The format in github changes but it's fine in our files, 2 space indentations on all files.

Reply:
Thanks we will work on it, we forgot to use editorconfig, and there seems to be tabs inserted

![Image 2021-05-14 at 8 29 44 p m](https://user-images.githubusercontent.com/11895273/118314122-f9724400-b4f3-11eb-94b3-d14ce4fff261.jpg)